gcp_variant_transforms/libs/processed_variant_test.py

```diff
@@ -466,5 +466,25 @@ def test_create_processed_variant_annotation_alt_allele_num(self):
         counter_factory.counter_map[
             CEnum.ALLELE_NUM_INCORRECT.value].get_value(), 4)
 
+  def test_create_alt_bases_field_schema_descriptions(self):
+    _, header_fields = self._get_sample_variant_and_header_with_csq()
+    for hfi in header_fields.infos.values():
+      hfi['type'] = 'string'
 
-# TODO(bashir2): Add tests for create_alt_record_for_schema.
+    counter_factory = _CounterSpyFactory()
+    factory = processed_variant.ProcessedVariantFactory(
+        header_fields,
+        split_alternate_allele_info_fields=True,
+        annotation_fields=['CSQ'],
+        counter_factory=counter_factory)
```

Comment:
Since you are not using `counter_factory` in this test, you can simply remove this line and drop `counter_factory` above too.